prombench/manifests/funcbench/benchmark/2_job.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: funcbench-test-{{ .PR_NUMBER }}
+  namespace: funcbench-{{ .PR_NUMBER }}
+spec:
+  # never re-create a new pod
+  backoffLimit: 0
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: funcbench
+        image: docker.io/prominfra/funcbench:master
+        imagePullPolicy: Always
+        args:
+          - "--verbose"
+          - "--owner"
+          - "{{ .GITHUB_ORG }}"
+          - "--repo"
+          - "{{ .GITHUB_REPO }}"
+          - "--github-pr"
+          - "{{ .PR_NUMBER }}"
+          - "--workspace"
```

Comment:
can you remind me what is this arg and why is it needed?

Reply:
`workspace` is a replacement of env variable `GITHUB_WORKSPACE`, it's the directory where the source will be cloned to.

Reply:
can you give me an example of why do we need to have this as an argument and why can't we always use the GITHUB_WORKSPACE env?

Reply:
It's a change introduced by another [pull request](https://github.com/prometheus/test-infra/pull/346#issue-401923405). In this PR, we update the example based on the change.

Reply:
Or, are you suggesting `--workspace ${GITHUB_WORKSPACE}` ?

Reply:
no I mean do you even need to set this? Why not have some default so clone to `/tmp` and that is it?

I mean the flag can stay in the funcbench tool as an option if someone needs to set it something different than the default, but why can't see skip setting it in the workflow file and leave it to some default option??

Reply:
I see that the default is `/tmp/funcbench` so why do you need to set it to `/build` ?